device-types/FS/S3150-8T2FP.yaml

```diff
@@ -0,0 +1,40 @@
+---
+manufacturer: FS
+model: S3150-8T2FP
+slug: s3150-8t2fp
+part_number: 'S3150-8T2FP (#90130)'
+is_full_depth: false
+u_height: 1
+airflow: left-to-right
+comments: 'S3150-8T2FP, 8-Port Gigabit Ethernet L2+ PoE+ Switch, 8 x PoE+ Ports @ 130W, with 2 x 1Gb SFP, Fanless'
+console-ports:
+  - name: Console
+    type: rj-45
+    label: Console
+    description: Serial Console Port
+power-ports:
+  - name: Power Port 1
```

Comment:
In the configuration, is the PSU referred to as `Power Port 1`? If so then this is fine. Otherwise please make this match what the configuration/device refers to it as. 

Just a thought, if the PSU is not defined anywhere in the config/device cli maybe use `PSU1` since that is a bit more standard for naming.

Reply:
It's not mentioned more than as "Power Supply (AC)" - so I'll rename it to PSU1 for standard purposes.